src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.macios.cs

```diff
@@ -49,21 +49,32 @@ public override void ViewDidLayoutSubviews()
 
 		View.Superview.Layer.CornerRadius = 0.0f;
 		View.Superview.Layer.MasksToBounds = false;
-		PresentationController?.ContainerView?.Subviews.OfType<UIImageView>().FirstOrDefault()?.RemoveFromSuperview();
+
+		SetShadowView(PresentationController!.ContainerView);
 
 		SetElementSize(new Size(View.Bounds.Width, View.Bounds.Height));
 	}
 
-	/// <inheritdoc/>
-	public override void ViewDidDisappear(bool animated)
+	void SetShadowView(UIView target)
 	{
-		if (ViewController?.View is UIView view)
+		if (target.Class.Name == "_UICutoutShadowView")
```

Comment:
I would try to avoid such magic strings as they may be changed in feature ios versions.
can we somehow get that specific view?

Reply:
@VladislavAntonyuk , It seems that the target class is defined in UIKitCore, but I can't reference it.
Therefore, I decided to discriminate by Class.Name.
Sorry, I don't know any other way.